build/lib/extensions.ts

```diff
@@ -74,7 +73,7 @@ export function fromLocal(extensionPath: string, forWeb: boolean): Stream { // {
 			delete data.dependencies;
 			delete data.devDependencies;
 			if (data.main) {
-				data.main = data.main.replace('/out/', /dist/);
+				data.main = data.main.replace('/out/', '/dist/');
```

Comment:
Just to confirm - you took this directly from VS Code? If there's any changes you made yourself then make sure to add SQL CARBON EDIT tags.

Reply:
Sort of... I compared the files and took the changes in that were not within SQL Carbon Edit tags.